@cadsit  this fails on windows 

**MicrosoftOpenJdk ("11.0.13.8.1");**

```
15:46:43 VRB] Refresh Android Sdk Info
[15:46:43 DBG] Checking to see if requested path C:\Program Files\Microsoft\microsoft-11.jdk matches C:\Program Files\Microsoft ADDLOCAL=FeatureMain\. ShouldRegisterJdk: True
[15:46:43 INF] Microsoft OpenJDK 11.0.13.8.1: ExecutingAction -> NotProvisioned
[15:46:43 DBG] Adding clean-provisioning (originally clean-provisioning) to telemetry
[15:46:43 ERR] Error
Xamarin.Provisioning.ProvisioningException: Microsoft OpenJDK 11.0.13.8.1: provisioning claims to have completed successfully, but its condition still fails.
   at Xamarin.Provisioning.Provisionator.ProvisionItemAsync(Item item, CancellationToken cancellationToken) in /Users/runner/work/1/s/Provisionator/Provisionator.cs:line 320
   at Xamarin.Provisioning.Provisionator.ExecuteQueueAsync(CancellationToken cancellationToken) in /Users/runner/work/1/s/Provisionator/Provisionator.cs:line 116
   at Xamarin.Provisioning.Entry.MainAsync(String[] args) in /Users/runner/work/1/s/Provisionator/Entry.cs:line 290
##[error]The process 'D:\a\_work\_temp\c0ba6d35-c41b-42f9-9040-f06d41e555aa/provisionator.exe' failed with exit code 104
```

--- a/eng/provisioning/provisioning.csx
+++ b/eng/provisioning/provisioning.csx
@@ -1,5 +1,8 @@
-ForceJavaCleanup();
-MicrosoftOpenJdk ("11.0.13.8.1");
+if (IsMac)
+{
+	ForceJavaCleanup();
+	MicrosoftOpenJdk ("11.0.13.8.1");
+}
 
 string ANDROID_API_SDKS = Environment.GetEnvironmentVariable ("ANDROID_API_SDKS");
 

